src/publishing/service.ts

```diff
@@ -192,7 +192,9 @@ export class FedifyPublishingService implements PublishingService {
         ];
 
         const fedifyNote = new FedifyNote({
-            id: this.uriBuilder.buildObjectUri(FedifyNote, uuidv4()),
+            id:
+                note.apId ||
+                this.uriBuilder.buildObjectUri(FedifyNote, uuidv4()),
```

Comment:
This change is to have the same apId for the note published and the note in our posts table.